src/languages/en.ts

```diff
@@ -3132,7 +3133,19 @@ export default {
         missingCategory: 'Missing category',
         missingComment: 'Description required for selected category',
         missingTag: ({tagName}: ViolationsMissingTagParams) => `Missing ${tagName ?? 'tag'}`,
-        modifiedAmount: 'Amount greater than scanned receipt',
+        modifiedAmount: ({type, displayPercentVariance}: ViolationsModifiedAmountParams): string => {
+            switch (type) {
+                case 'distance':
+                    return 'Amount differs from calculated distance';
+                case 'card':
+                    if (displayPercentVariance) {
+                        return `Amount ${displayPercentVariance}% greater than scanned receipt`;
+                    }
+                    return 'Amount greater than card transaction';
+                default:
+                    return 'Amount greater than scanned receipt';
+            }
+        },
```

Comment:
https://github.com/Expensify/App/issues/41401#issuecomment-2091898713

>   * `{data: {type: 'distance'}}`
>   * `{data: {type: 'card'}}`
>   * `{data: {type: 'smartscan'}}`
>   * `{data: {type: 'smartscan', displayPercentVariance: $displayPercentVariance}}`

`displayPercentVariance` is present in type 'smartscan' not 'card'